manifests/local/storage-modules.yaml

```diff
@@ -0,0 +1,19 @@
+name: buckets
+path: git::https://github.com/awslabs/idf-modules.git//modules/storage/buckets?ref=release/1.0.0&depth=1
+targetAccount: primary
+targetRegion: us-east-1
+parameters:
+  - name: encryption-type
+    value: SSE
+  - name: retention-type
+    value: RETAIN
+---
+name: opensearch
+path: git::https://github.com/awslabs/idf-modules.git//modules/storage/opensearch?ref=release/1.0.0&depth=1
```

Comment:
If you want to reference the git release, then these manifests should be in the examples dir....
If you want the path to be a local path, then these manifests would make sense

We either need to make the path relative or remove these manifests